metabase/metabase
# Always create new items in the current collection

Product doc https://www.notion.so/metabase/Make-Metabase-realize-I-m-trying-to-do-work-and-help-me-out-51ada72df7f94da483a0cc8c30cc173f?d=31d8a44a318a4322b184f8e4560cde2d#a02efa1d6a7640d8b42b61aebf7f5e5d

How to test:
- Open a non-root collection
- Click `New` -> `Collection`
- Make sure the suggested collection is the opened one

<img width="1359" alt="Screenshot 2022-08-25 at 14 18 33" src="https://user-images.githubusercontent.com/8542534/186650909-cf268799-364c-4246-8ba9-c90020c87cb8.png">
